CHANGELOG.rst

```diff
@@ -9,6 +9,8 @@ Changelog
 * The :class:`~cryptography.x509.Certificate` class now has
   :attr:`~cryptography.x509.Certificate.signature` and
   :attr:`~cryptography.x509.Certificate.tbs_certificate_bytes` attributes.
+* :class:`~cryptography.x509.NameConstraints` are now supported in the
+  :class:`~cryptography.x509.CertificateBuilder`.
```

Comment:
Some users might care (sometimes internal CAs allow issuance using extensions requested in the CSR), so I'll go ahead and mention CSRBuilder supports it too.